### DISCUSSION TITLE
用python進行股票分析

### DISCUSSION THREAD

**Discussion by wu51000** [1101-GS3073-C/Python-Community-GS3073-C#735] [General]
雖然已經有許多相關模組了 但還是想分享給不知道的同學！
憑著小資族要學會投資理財的精神，查找了如何用python分析股市股價的網站。
沒想到真的有很多很厲害的分析工具、模組…..長知識了…..
當然雞蛋不能放同個籃子裡，但若有好工具的輔助可以幫助我們事半功倍！
（連結不知為何點不進去……我補個新在這！）
https://pyradise.com/用-python-與-excel-打造簡易的回測系統-857bfbb88867

**Comment by yjpxi** [comment]
我覺得如果是我要買股票會選擇ai來幫我分析而不是股票分析師，因為ai本身就擁有大量資料庫，也可以看未來的趨勢，可以很快的幫你製作股票的圖表。
